sdk/adcampaign/adcampaignactivity.go

```diff
@@ -0,0 +1,59 @@
+package adcampaign
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mazti/facebook-go-business-sdk/sdk"
+)
+
+type AdCampaignActivity struct {
+	node          *sdk.APINode
+	ID            string `json:"id"`
+}
+
+func NewAdCampaignActivity(id string, context *sdk.APIContext) *AdCampaignActivity {
+	return &AdCampaignActivity{
+		node: sdk.CreateAPINode(context),
+		ID:   id,
+	}
+}
+func ParserResponse(data json.RawMessage) (sdk.APIResponse, error) {
+	ent := &AdCampaignActivity{}
+	if err := json.Unmarshal(data, ent); err != nil {
+		return ent, err
+	}
+	return ent, nil
+}
+
+func FetchByID(id string, context *sdk.APIContext) (*AdCampaignActivity, error) {
+	req := sdk.NewAPIRequest(
+		context,
+		id,
+		sdk.DefaultEndpoint,
+		http.MethodGet,
+		sdk.Parser(ParserResponse),
```

Comment:
Try to use  sdk.ReturnFields(fields)
`
	req := sdk.NewAPIRequest(
		context,
		id,
		sdk.DefaultEndpoint,
		http.MethodGet,
		sdk.Parser(parserResponse),
		sdk.ReturnFields(fields),
	)
`
For get some fields we need on facebook